src/strings/i18n.js

```diff
@@ -0,0 +1,23 @@
+import i18n from 'i18next';
+import XHR from 'i18next-xhr-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import english from './english'
+
+i18n
+  .use(XHR)
```

Comment:
Delete this and uninstall `i18next-xhr-backend` - we're bundling the translations with the app by importing them, not loading them asynchronously.